pkg/server/plugin/ca/memory/memory.go

```diff
@@ -221,51 +215,51 @@ func (m *MemoryPlugin) LoadCertificate(ctx context.Context, request *ca.LoadCert
 	}
 
 	if len(uris) != 1 {
-		return &ca.LoadCertificateResponse{}, fmt.Errorf("X.509 SVID certificates must have exactly one URI SAN. Found %v URI(s)", len(uris))
+		return &ca.LoadCertificateResponse{}, fmt.Errorf("load certificate: found %v URI(s); must have exactly one", len(uris))
```

Comment:
one of these days we should discuss introducing something like github.com/zeebo/errs to the codebase to cut down on some of the verbosity around error creation (and get other benefits like stack traces).